MediaBrowser.Plugins.Trailers/Providers/ML/BaseProvider.cs

```diff
@@ -291,9 +292,9 @@ private List<ChannelMediaInfo> GetMediaInfo(IEnumerable<HtmlNode> nodes, string
 
                 var url = subString.Substring(0, endIndex);
 
-                //_logger.Debug("Found url: " + url);
+                //_logger.LogDebug("Found url: {url}", url);
```

Comment:
You probably did a search and replace, but maybe remove this line

Reply:
removing